Compilation failure with text-2.1.2

`Data.Text.Lazy` now exports `show` causing `Network` to fail to build:

```
[ 1 of 19] Compiling InputParser      ( src/InputParser.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/InputParser.o )
[ 2 of 19] Compiling Network          ( src/Network.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Network.o )
src/Network.hs:14:54: error: [GHC-87543]
    Ambiguous occurrence ‘show’.
    It could refer to
       either ‘Prelude.show’,
              imported from ‘Prelude’ at src/Network.hs:1:8-14
              (and originally defined in ‘ghc-internal-9.1003.0:GHC.Internal.Show’),
           or ‘Data.Text.Lazy.show’,
              imported from ‘Data.Text.Lazy’ at src/Network.hs:8:1-21.
   |
14 |                 putStrLn $ "load " ++ url ++ " (" ++ show retries ++ " more attempts)..."
   |                                                      ^^^^

[ 3 of 19] Compiling Paths_adblock2privoxy ( dist/build/adblock2privoxy/autogen/Paths_adblock2privoxy.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Paths_adblock2privoxy.o )
[ 4 of 19] Compiling PopupBlocker     ( src/PopupBlocker.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/PopupBlocker.o )
[ 5 of 19] Compiling ProgramOptions   ( src/ProgramOptions.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/ProgramOptions.o )
[ 6 of 19] Compiling SourceInfo       ( src/SourceInfo.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/SourceInfo.o )
[ 7 of 19] Compiling Statistics       ( src/Statistics.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Statistics.o )
[ 8 of 19] Compiling Task             ( src/Task.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Task.o )
[ 9 of 19] Compiling Utils            ( src/Utils.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Utils.o )
[10 of 19] Compiling PolicyTree       ( src/PolicyTree.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/PolicyTree.o )
[11 of 19] Compiling ParsecExt        ( src/ParsecExt.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/ParsecExt.o )
[12 of 19] Compiling PatternConverter ( src/PatternConverter.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/PatternConverter.o )
[13 of 19] Compiling UrlBlocker[boot] ( src/UrlBlocker.hs-boot, dist/build/adblock2privoxy/adblock2privoxy-tmp/UrlBlocker.o-boot )
[14 of 19] Compiling Templates        ( src/Templates.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/Templates.o )
src/Templates.hs:6:1: warning: [GHC-66111] [-Wunused-imports]
    The import of ‘Data.Foldable’ is redundant
      except perhaps to import instances from ‘Data.Foldable’
    To import instances alone, use: import Data.Foldable()
  |
6 | import Data.Foldable
  | ^^^^^^^^^^^^^^^^^^^^

[15 of 19] Compiling OptionsConverter ( src/OptionsConverter.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/OptionsConverter.o )
src/OptionsConverter.hs:97:32: warning: [GHC-63394] [-Wx-partial]
    In the use of ‘head’
    (imported from Data.List, but defined in GHC.Internal.List):
    "This is a partial function, it throws an error on empty lists. Use pattern matching, 'Data.List.uncons' or 'Data.Maybe.listToMaybe' instead. Consider refactoring to use "Data.List.NonEmpty"."
   |
97 |                 code' = sort $ head . dropWhile (`elem` "/(?:x-)") <$> contentTypes'
   |                                ^^^^

[16 of 19] Compiling UrlBlocker       ( src/UrlBlocker.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/UrlBlocker.o )
[17 of 19] Compiling ParserExtTests   ( src/ParserExtTests.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/ParserExtTests.o )
[18 of 19] Compiling ElementBlocker   ( src/ElementBlocker.hs, dist/build/adblock2privoxy/adblock2privoxy-tmp/ElementBlocker.o )
src/ElementBlocker.hs:43:32: warning: [GHC-63394] [-Wx-partial]
    In the use of ‘head’
    (imported from Data.List, but defined in GHC.Internal.List):
    "This is a partial function, it throws an error on empty lists. Use pattern matching, 'Data.List.uncons' or 'Data.Maybe.listToMaybe' instead. Consider refactoring to use "Data.List.NonEmpty"."
   |
43 |            if isDir then when (head entry' /= '.') $ removeDirectoryRecursive entry
   |                                ^^^^
```

--- a/adblock2privoxy-utils/adblock2privoxy-utils.cabal
+++ b/adblock2privoxy-utils/adblock2privoxy-utils.cabal
@@ -1,5 +1,5 @@
 name:           adblock2privoxy-utils
-version:        2.3.0
+version:        2.3.1
 cabal-version:  >= 1.10
 build-type:     Simple
 tested-with:    GHC==8.10.7
@@ -45,4 +45,4 @@ source-repository this
   type:         git
   location:     https://github.com/essandess/adblock2privoxy.git
   subdir:       adblock2privoxy-utils
-  tag:          v2.3.0
+  tag:          v2.3.1

--- a/adblock2privoxy/CHANGELOG.md
+++ b/adblock2privoxy/CHANGELOG.md
@@ -1,3 +1,5 @@
+2.3.1
+	* Fix Network.hs compilation
 2.3.0
 	* hlint mods
 	* Add debug code with new DebugLevel option

--- a/adblock2privoxy/adblock2privoxy.cabal
+++ b/adblock2privoxy/adblock2privoxy.cabal
@@ -1,9 +1,9 @@
 name:           adblock2privoxy
-version:        2.3.0
+version:        2.3.1
 cabal-version:  >= 1.10
 build-type:     Simple
 tested-with:
-                GHC==9.6.3
+                GHC==9.12.2
 author:         Alexey Zubritsky <adblock2privoxy@zubr.me>, Steven Thomas Smith <steve.t.smith@gmail.com>
 data-files:
                 templates/ab2p.system.action,
@@ -104,4 +104,4 @@ source-repository this
   type:         git
   location:     https://github.com/essandess/adblock2privoxy.git
   subdir:       adblock2privoxy
-  tag:          v2.3.0
+  tag:          v2.3.1

--- a/adblock2privoxy/src/Network.hs
+++ b/adblock2privoxy/src/Network.hs
@@ -5,7 +5,7 @@ where
 
 import Network.HTTP.Conduit
 import Data.Text.Lazy.Encoding
-import Data.Text.Lazy
+import Data.Text.Lazy (unpack)
 import Control.Exception
 
 -- | A simpleHttp alternative that specifies bigger timeout and retries connection attempts

--- a/adblock2privoxy/src/Paths_adblock2privoxy.hs
+++ b/adblock2privoxy/src/Paths_adblock2privoxy.hs
@@ -38,7 +38,7 @@ catchIO :: IO a -> (Exception.IOException -> IO a) -> IO a
 catchIO = Exception.catch
 
 version :: Version
-version = Version [2,3,0] []
+version = Version [2,3,1] []
 bindir, libdir, dynlibdir, datadir, libexecdir, sysconfdir :: FilePath
 
 bindir     = "/opt/local/bin"

